%%%%%%%%%%%%%%
%% PREAMBLE %%
%%%%%%%%%%%%%%

\documentclass[11pt, letterpaper, sans]{moderncv}
\moderncvstyle{classic}
\moderncvcolor{grey}
\renewcommand{\familydefault}{\sfdefault}
\usepackage{tikz}
\usepackage{xcolor}
\usepackage{multicol}
\usepackage{fontawesome}
\usepackage[utf8]{inputenc}
\usepackage{hanging}
\usepackage{geometry}
\geometry{left=1.5cm, right=1.5cm, top=0.5cm, bottom=1.5cm, heightrounded}

%%%%%%%%%%%%%%%%%%%
%% PERSONAL DATA %%
%%%%%%%%%%%%%%%%%%%

\firstname{Denis} \familyname{Jacob~Machado,~Ph.D.}
\address{}{}{UNC Charlotte. Charlotte--NC, USA}
\email{dmachado@charlotte.edu}
\homepage{https://phyloinformatics.com}
\mobile{+1 (980) 287-9452}

%%%%%%%%%%%%%%%%%%%%%%%
%% HEADER AND FOOTER %%
%%%%%%%%%%%%%%%%%%%%%%%

\usepackage{xcolor}
\usepackage{etoolbox}
\usepackage{fancyhdr}
\definecolor{headercolor}{rgb}{0.35,0.35,0.35} % dark grey
\pagestyle{fancy}
\fancypagestyle{plain}{\fancyhf{}}
\renewcommand{\headrulewidth}{0pt}
\headheight 15pt
\footskip 45pt
\renewcommand{\footrulewidth}{0pt}
\newcommand{\helv}{
\fontsize{8}{10}\selectfont}
\makeatletter
\patchcmd{\@fancyhead}{\rlap}{\color{headercolor}\rlap}{}{}
\patchcmd{\headrule}{\hrule}{\color{headercolor}\hrule}{}{}
\patchcmd{\@fancyfoot}{\rlap}{\color{headercolor}\rlap}{}{}
\patchcmd{\footrule}{\hrule}{\color{headercolor}\hrule}{}{}
\makeatother
\fancyfoot[L]{\color{headercolor}\nouppercase{\helv\small{}}}
\fancyfoot[R]{\color{headercolor}\nouppercase{\helv\small{}}}
\nopagenumbers{}

\begin{document}\thispagestyle{empty}\maketitle

%%%%%%%%%%%%%%%%
%% HIGHLIGHTS %%
%%%%%%%%%%%%%%%%

\vspace{-3em}

\textbullet~\textbf{Bioinformaticist} with strong background on invertebrate zoology, molecular biology, and bioinformatics.

\textbullet~Peer-reviewed publications on topics including \textbf{genomics}, \textbf{evolution}, \textbf{parasitology}, \textbf{regeneration}, \textbf{emerging infectious diseases}, and \textbf{genomic epidemiology}.

\textbullet~$5+$ years of experience using \textbf{machine learning} (e.g., PyTorch, TensorFlow), \textbf{predictive modeling}, \textbf{high-throughput data processing}, and \textbf{data mining} algorithms to solve practical problems in biology.

\textbullet~$13+$ years of experience in \textbf{molecular biology}, \textbf{high-throughput sequencing} (e.g., Illumina, PacBio, Hi-C, GWAS, and RNA-Seq), and sequence assembly, and differential gene expression analyses.

%%%%%%%%%%%%%%%%%%%%
%% POSITIONS HELD %%
%%%%%%%%%%%%%%%%%%%%

\vspace{-0.5em}

\section{Work Experience}
\cventry{2019--2022}
{Assistant Professor in Bioinformatics}
{University of North Carolina at Charlotte}
{Charlotte--NC}
{USA}
{
\textbullet~Investigate new pathogens' emergence, evolution, and spread focusing on preventing and treating infectious diseases.\\
\textbullet~Create computational and molecular solutions to make data from biorepositories more readily available to biomedical research.\\
}
%
\cventry{2019--2022}
{Postdoctoral Researcher}
{University of North Carolina at Charlotte}
{Charlotte--NC}
{USA}
{
\textbullet~I conducted the most diverse and complete analysts of the evolution of coronaviruses (e.g., SARS-CoV, MERS-CoV, and SARS-CoV-2) and flaviviruses (e.g., dengue, zika, and west-nile virus)
\textbullet~Produced draft genomes of highly regenerative echinoderms such as brittle stars and sea cucumbers for
comparative genomics and tissue regeneration research
}
%
\cventry{2018--2019}
{Research Collaborator}
{University of São Paulo}
{São Paulo--SP}
{Brazil}
{
\textbullet~I built and administered the first computer clusters at the Muzeum of Zoology of the University of São Paulo (MUZUSP), providing training and technology to bridge the gap between basic animal research and applied bioinformatics with potential biomedical significance.\\
\textbullet~I participated in the planning, funding acquisition, and implementation of the first laboratory to sequence historical DNA (e.g., from degraded museum samples) at the Department of Zoology of the University of São Paulo (USP).\\
\textbullet~I developed new genome skimming techniques to retrieve genomic data of non-model organisms when samples were small or degraded. I also created the necessary software to facilitate the analysis of organelle genomes, including new indices that check for the completeness of circular genomes or misalignments in other types of contigs and scaffolds.
}
%
\cventry{2018--2019}
{Invited Lecturer}
{University of São Paulo}
{São Paulo--SP}
{Brazil}
{
\textbullet~I created and taught the first graduate-level courses in bioinformatics and computer programming for biologists at USP's Graduate Program in Zoology, which is ranked first in animal research worldwide.
}
%
\cventry{Jul--Aug 2017}
{Invited Lecturer}
{University of Magdalena}
{Santa Marta D.T.C.H.}
{Colombia}
{
\textbullet~I designed and taught the first course in bioinformatics at the University of Magdalena. Classes included practical and theoretical lessons on DNA extraction and isolation, high-throughput sequencing, sequence alignment, reference, and \textit{de novo} genome assembly, and gene prediction and annotation.\\
\textbullet~I also offered consultation to faculty members involved in animal research who needed web and dry lab solutions for next-generation sequence analyses.
}

%%%%%%%%%%%%%%%%%%%%%%
%% FORMAL EDUCATION %%
%%%%%%%%%%%%%%%%%%%%%%

\vspace{-1em}

\section{Education}

\cventry{2013--2018}
{Ph.D. in Bioinformatics}
{University of São Paulo (USP)}
{São Paulo--SP}
{Brazil}
{}

\cventry{2010--2012}
{Master's in Zoology (Parasitology)}
{University of São Paulo (USP)}
{São Paulo--SP}
{Brazil}
{}

\cventry{2006--2009}
{Bachelor's in Biological Sciences (Marine Biology)}
{São Paulo State University (UNESP)}
{São Vicente--SP}
{Brazil}
{}

Visit my profile on Google Scholar at \url{https://scholar.google.com/citations?user=TK6Ms80AAAAJ&hl=pt-BR}.\\
You will find my \textit{curriculum vitae} and other professional information at \url{https://phyloinformatics.com}.

\end{document}
